test/operator-hub/catalog_source.yaml

```diff
@@ -6,6 +6,6 @@ metadata:
   namespace: openshift-operator-lifecycle-manager
 spec:
   sourceType: grpc
-  image: REPLACE_IMAGE
+  image: quay.io/redhat-developer/app-binding-operator:4c0a725
```

Comment:
Why tag is hard-coded here?

Reply:
I don't see this file is being used in the Makefile. Do we really need the catalog source?